src/components/pages/events/containers/SubmittedPageContainer.js

```diff
@@ -0,0 +1,9 @@
+import { compose } from 'recompose';
+import { connect } from 'react-redux';
+import SubmittedPage from '../SubmittedPage';
+
+export default compose(
+  connect(state => ({
+    submittedEvent: state.event.submittedEvent
```

Comment:
I guess all events are submitted via redux so this can never be an old value when user gets to submitted page?